tests/reporter/kubernetes.go

```diff
@@ -190,18 +198,15 @@ func (r *KubernetesReporter) dumpNamespaces(duration time.Duration, vmiNamespace
 	r.logSecrets(virtCli)
 	r.logNetworkAttachmentDefinitionInfo(virtCli)
 	r.logKubeVirtCR(virtCli)
-	r.logNodes(virtCli, nodes)
-	r.logPods(virtCli, pods)
+	r.logNodes(nodes)
+	r.logPods(pods)
 	r.logVMs(virtCli)
-	r.logVMSnapshot(virtCli)
```

Comment:
Where is this duplicated?

Reply:
Actually, this was duplicate `r.logVirtualMachineSnapshots(virtCli)`